css/base/_base.css

```diff
@@ -19,30 +19,25 @@ html {
 
 html,
 body {
-	margin: 0;
-	padding: 0;
+  margin: 0;
+  padding: 0;
 }
 
 body {
-	background: $vintage-background;
-	font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+  background: $vintage-background;
+  font-family: $ui-font-stack;
 }
 
 .wrapper {
-	color: #222222;
-	font-family: monospace;
-	max-width: $large-width;
-	margin: 0 auto;
-	white-space: pre-wrap;
-}
-
-body.js-source-code-pro-loaded .wrapper {
```

Comment:
If we are removing Source Code Pro completely there was some logic in app.js that can now be removed.